dockstore-webservice/src/main/java/io/dockstore/webservice/core/database/VersionVerifiedPlatform.java

```diff
@@ -0,0 +1,44 @@
+package io.dockstore.webservice.core.database;
+
+/**
+ * This class is a subset of fields from sourcefile and verificationinformation. States what platform (source) a version has been verified on
+ * and who performed the verification (metadata).
+ * @author natalieperez
+ * @since 1.10.0
+ */
+public class VersionVerifiedPlatform {
+
+    private long versionId;
+    private String metadata;
+    private String source;
+
+    public VersionVerifiedPlatform(long versionId, String source, String metadata) {
+        this.versionId = versionId;
+        this.source = source;
+        this.metadata = metadata;
+    }
+
+    public long getVersionId() {
+        return versionId;
+    }
+
+    public void setVersionId(final long versionId) {
```

Comment:
I don't think you need setters. These objects are created via the named query and are not modified after.

If I'm right, I would get rid of the setters and make the fields final. Also, the parameters in the constructor should probably be final too -- doesn't Codacy flag that one?

In general, immutability is good.